code/modules/jobs/job_types/shaft_miner.dm

```diff
@@ -1,3 +1,35 @@
+GLOBAL_LIST_INIT(miner_callouts, list(
+	/obj/item/stack/sheet/mineral/gold = list(":H We're rich!"),
+	/obj/item/stack/ore/gold = list(":H Gold!", ":H There is gold here!", ":H There is gold!", ":H I found a gold vein!"),
+	/obj/structure/flora/ash/leaf_shroom = list(":H Mushroom!"),
+	/obj/structure/flora/ash/cap_shroom = list(":H Mushroom!"),
+	/obj/structure/flora/ash/stem_shroom = list(":H Mushroom!"),
+	/obj/structure/closet/crate/secure/loot = list(":H Abandoned crate here!", ":H Found a crate with a codelock over here!"),
+	/obj/structure/closet/crate/necropolis = list(":H Spooky chest here, someone got a key?", ":H Necropolis chest here!"),
+	/obj/structure/spawner/lavaland = list(":H Tendril spotted!", ":H Got a tendril here!", ":H Found a tendril!"),
+	/obj/structure/geyser = list(":H Got a geyser here, those chemists'll be pleased!", ":H Found a geyser!", ":H Geyser here!"),
+	/mob/living/simple_animal/hostile/asteroid/basilisk = list(":H Basilisk, watch out!", ":H Spotted a Basilisk!", ":H Look out for that Basilisk!"),
```

Comment:
Forgive me if this is intended, but wouldn't this be more useful if it was focused on watchers? Basilisks don't really spawn much at all (except in xenobio), but watchers (a subtype) do.

Reply:
`...asteroid/basilisk/watcher`